src/20240220_AaveV3Ethereum_OrbitProgram/OrbitProgramData.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+library OrbitProgramData {
+  struct GasUsage {
+    address account;
+    uint256 usage;
+  }
+
+  uint256 public constant STREAM_DURATION = 90 days;
+  uint256 public constant RETRO_PAYMENT = 5_000 ether;
+  uint256 public constant STREAM_AMOUNT = 15_000 ether;
+  uint256 public constant TOTAL_WETH_REBATE = 4.9273 ether;
+  address public constant EZREAL = 0x8659D0BB123Da6D16D9394C7838BA286c2207d0E;
+  address public constant STABLE_LABS = 0xECC2a9240268BC7a26386ecB49E1Befca2706AC9;
+  address public constant LBS = 0x8b37a5Af68D315cf5A64097D96621F64b5502a22;
+  address public constant MICHIGAN = 0x0579A616689f7ed748dC07692A3F150D44b0CA09;
```

Comment:
Michigan address is different than the one on the forum post table

Reply:
added to AIP and it's here: https://governance.aave.com/t/arfc-orbit-program-renewal/16550/6